src/components/RichTextContainer.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, createElement } from "react";
+
+import { CommonRichTextProps, RichText } from "./RichText";
+import { ValidateConfigs } from "./ValidateProps";
+
+import { getValue, parseStyle } from "../utils/ContainerUtils";
+
+interface WrapperProps {
+    "class": string;
+    mxform: mxui.lib.form._FormBase;
+    mxObject?: mendix.lib.MxObject;
+    style: string;
+    readOnly: boolean;
+}
+
+export interface RichTextContainerProps extends WrapperProps, CommonRichTextProps {
+    stringAttribute: string;
+    editable: "default" | "never";
+    onChangeMicroflow: string;
+}
+
+interface RichTextState {
+    value: string;
+}
+
+export type ReadOnlyStyle = "bordered" | "text" | "borderedToolbar";
+
+export default class RichTextContainer extends Component<RichTextContainerProps, RichTextState> {
+    private subscriptionHandles: number[] = [];
+
+    constructor(props: RichTextContainerProps) {
+        super(props);
+
+        this.state = {
+            value: String(getValue(props.stringAttribute, "", props.mxObject))
```

Comment:
`as string` doesn't cast & I needed the cast. Didn't know` <string>` does casting... Nice.